pyls/plugins/pydocstyle_lint.py

```diff
@@ -19,8 +22,10 @@ def pyls_settings():
 @hookimpl
 def pyls_lint(document):
     conf = pydocstyle.config.ConfigurationParser()
-    conf.parse()
-    conf._arguments = [document.path]
+
+    with _patch_sys_argv([document.path]):
+        # TODO(gatesn): We can add more pydocstyle args here from our pyls config
+        conf.parse()
```

Comment:
We can also add the options `ignore` and `select` which are ignored when set on the user configuration file: 
```javascript
"pyls": {
    "plugins": {
        "pydocstyle": {
            "enabled": true,
            "ignore": [
                "D202",
                "D208"
            ]
        }
    }
}
```

Reply:
follow-up in #249